tests/readyflag/impl/ReadyFlagCppImplementation.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include "tests/readyflag/ReadyFlagInterfacePropertyAdapter.h"
```

Comment:
ok, sorry, I did not mean to remove all empty lines, you should of course use empty lines to group things appropriately. Please take some other files as a reference.